justfile

```diff
@@ -42,6 +42,8 @@ DC_USER := env_var_or_default("DC_USER", "opener")
 # Installs Node.js dependencies for the entire monorepo.
 node-install:
     pnpm i
+    just frontend/run i18n:en
+    just frontend/run i18n:copy-test-locales
```

Comment:
Will we run `node-install` only in CI? Why are we copying test locales here?

Reply:
We run this in dev and CI by running `just install` (basically everywhere except the Docker container).

Reply:
Can we add a note somewhere in the docs that the locales in dev and the CI are the "test" locales, and not the actual locales that are used on the production site?

Reply:
Oh, if we are using the test locales in dev, too, we can stop calling the other i18n locale scripts in both dev and CI. They are not used anyways, this will lower the number of outgoing requests.

Reply:
I didn't understand what you mean. In the interest of time, feel free to push your changes to this PR.